client/api/contentElement.js

```diff
@@ -0,0 +1,13 @@
+import request from './request';
+
+const urls = {
+  resource: (repoId, id) => `/repositories/${repoId}/content-elements/${id}`
```

Comment:
Extract `/repositories/${repoId}/content-elements/` as root and use full name for repositoryId. Then use root for defining resource.